lib/linux_admin/common.rb

```diff
@@ -1,25 +1,25 @@
 require 'awesome_spawn'
 
 module LinuxAdmin
-  module Common
+  class Common
```

Comment:
Do you need to change this?  I don't see us creating an instance anywhere and all the methods are class methods.